examples/TransactionGenerator.hs

```diff
@@ -141,7 +142,7 @@ defaultTransactionConfig :: TransactionConfig
 defaultTransactionConfig =
   TransactionConfig
     { _scriptCommand      = DeployContracts []
-    , _nodeChainId        = unsafeChainId 0
+    , _nodeChainId        = someChainId Testnet00
```

Comment:
Should the chainweb version be configurable? If so, the default settings for `_serverRootPrefix` and `_chainwebNodePort` could be derived from the boots.trap nodes of the respective version.

Reply:
Almost certainly. @emmanueldenloye uses `TestWithTime` locally on his machine.